html/changelogs/DreamySkrell-loadout-qol-236.yml

```diff
@@ -0,0 +1,7 @@
+author: DreamySkrell
```

Comment:
duplicate changelog?

Reply:
Not duplicate. That is changelog from #14982, which needs to be merged before this PR.

Reply:
you don't need the changelog in this PR if you have it in the previous one.

Reply:
I need the changelog in this PR. The previous PR is open and awaiting merge. When that PR gets merged, this PR will have only one changelog.

Reply:
no, you don't need the duplicate changelog in this PR. it is handled in the other PR.

Reply:
This extra changelog will be automatically deleted once the other PR is merged. The only reason why this PR is here in the first place is because it's a carbon copy of the first Loadout QoL PR, and that's just the way Git works. This is all automatic.